internal/engine/rtengine/engine.go

```diff
@@ -177,7 +177,7 @@ func (r *RuleTypeEngine) Eval(
 
 	// Process evaluation
 	logger.Info().Msg("entity evaluation - evaluation started")
-	err := r.ruleEvaluator.Eval(ctx, params.GetRule().Def, result)
+	err := r.ruleEvaluator.Eval(ctx, params.GetRule().Def, inf.Entity, result)
```

Comment:
I could turn this into the same pattern as the one used [here](https://github.com/stacklok/minder/pull/4584) to limit the changes across the code base, let me know what you folks prefer.